Envelopes/cool_MK ShaperStutter.lua

```diff
@@ -1,14 +1,41 @@
 -- @description MK Shaper/Stutter
 -- @author cool
--- @version 1.01
+-- @version 1.20
```

Comment:
Did you mean 1.2 or 1.02? 1.20 ("twentieth minor update of major version 1") might be higher than you think.

Reply:
1.20 is ok. I've been making a lot of big changes to the code, but haven't found the time to publish it. That's why it's such a big leap.
Thank you!